src/constants/constants.js

```diff
@@ -1,3 +1,3 @@
-export const CHI_VERSION = '5.30.0';
+export const CHI_VERSION = '5.31.0';
```

Comment:
This will be push after the new Chi 5.32.0 release.

```suggestion
export const CHI_VERSION = '5.32.0';
```